src/app/migration/src/src/app/modules/announcement/components/outbox/outbox.component.html

```diff
@@ -0,0 +1,121 @@
+<div>
+  <div class="ui grid">
+    <div class="one wide column"></div>
+    <div class="ten wide column">
+      <div class="ui grid">
+        <div class="nine wide column">
+          <div class="ui">
+            <div class="text dashBoardSectionHeading padding-top-10"><i class="announcement icon padding-right-10"></i> {{resourceService?.frmelmnts?.lbl?.anncmntmine}}</div>
+          </div>
+        </div>
+        <div class="right floated three wide column padding-right-dropdown padding-right-10">
+          <div class="ui float-ContentRight">
+            <button class="ui primary basic button announcementButton"><span class="announcementButtonText">{{resourceService?.frmelmnts?.lbl?.anncmntcreate}}</span></button>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="ui grid" *ngIf="showDataDiv">
+    <div class="one wide column"></div>
+    <div class="ten wide column">
+      <div class="ui grid">
+        <div class="eight wide column">
+          <div class="ui">
+            <div class="text padding-top-5 showingAllSentAnn">{{resourceService?.frmelmnts?.lbl?.anncmntsent}}</div>
+          </div>
+        </div>
+        <div class="four wide column">
+          <div class="ui float-ContentRight">
+            <div class="text showingAllSentAnn"><span class="margin-right-10">Viewing {{pager.startIndex+1}} - {{pager.endIndex+1}} of  {{totalCount}} </span>
+              <div class="ui small basic icon buttons">
+                <button class="ui button" [ngClass]="{'disabled' : pager.startIndex === 0}" (click)="setPage(pager.currentPage - 1)"><i class="left chevron icon"></i></button>
+                <button class="ui button" [ngClass]="{'disabled' : pager.currentPage === pager.totalPages}" (click)="setPage(pager.currentPage + 1)"><i class="right chevron icon"></i></button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="ui grid" *ngIf="showLoader">
+    <div class="one wide column"></div>
+    <div class="ten wide column">
+      <app-loader></app-loader>
+    </div>
+  </div>
+  <div class="ui grid" *ngIf="!showDataDiv && !showLoader">
```

Comment:
remove showDataDiv and replace it with outBoxList.length === 0

Reply:
done